doc/source/whatsnew/v0.18.2.txt

```diff
@@ -315,6 +315,6 @@ Bug Fixes
 - Bug in ``groupby`` where ``apply`` returns different result depending on whether first result is ``None`` or not (:issue:`12824`)
 
 
-
+- Bug in series comparison operators when dealing with zero rank numpy arrays (:issue:`13006`)
 
```

Comment:
Use ``Series`` (surrounded by double-ticks) instead of "series" and change "numpy" to "NumPy"